docs/rules/jsx-fragments.md

```diff
@@ -58,6 +58,5 @@ Examples of **correct** code for this rule:
 <React.Fragment key="key"><Foo /></React.Fragment>
 ```
 
-[fragments]: https://reactjs.org/docs/fragments.html
```

Comment:
maybe instead of deleting this link, we should link to it on line 5?

Reply:
Good idea, fixed.